arduino/discovery/discovery.go

```diff
@@ -121,8 +121,9 @@ func (p *Port) String() string {
 
 // Event is a pluggable discovery event
 type Event struct {
-	Type string
-	Port *Port
+	Type        string
+	Port        *Port
+	DiscoveryID string
```

Comment:
reorder this please: https://medium.com/wesionary-team/simple-data-alignment-technique-to-speed-up-your-struct-in-golang-954c43faa1fe

Reply:
No.

The CLI is already complex on its own, I will not increase the cognitive load for gains that are negligible (save 2 bytes of RAM? really? :-)